mapfishapp/src/deb/resources/etc/georchestra/mapfishapp/js/GEOR_custom.js

```diff
@@ -0,0 +1,80 @@
+Ext.namespace("GEOR");
+
+GEOR.custom = {
+    HEADER_HEIGHT: 90,
```

Comment:
If `georchestra.datadir`:

It is the administrator's responsability to ensure his datadir is correctly configured, the main added value using the datadir is that the administrator is able to update the parameter without rebuilding the whole suite/webapp.

if `! georchestra.datadir`:
Then one will use the GEOR_custom from the build, so it would have the expected HEADER_HEIGHT.

So, I cannot see the point